app/src/main/java/mozilla/org/webmaker/activity/Page.java

```diff
@@ -7,4 +7,9 @@
     public Page() {
         super("page", R.id.page_layout, R.layout.page_layout, R.menu.menu_page);
     }
+
+    @Override
+    public void onBackPressed() {
```

Comment:
I could also move this `WebmakerActivity` to be more generic, but I figured it might be better to not mess around with the back button where not necessary. What do you think?